scripts/posteffects/posteffect-ssao.js

```diff
@@ -17,23 +17,25 @@ function SSAOEffect(graphicsDevice, ssaoScript) {
     this.ssaoScript = ssaoScript;
     this.needsDepthBuffer = true;
 
+    var vshader = [
+        (graphicsDevice.webgl2) ? ("#version 300 es\n\n" + pc.shaderChunks.gles3VS) : "",
+        graphicsDevice.webgl2 ? "" : "#extension GL_OES_standard_derivatives: enable\n",
+        graphicsDevice.webgl2 ? "" : "attribute vec2 aPosition;",
+        "",
+        "varying vec2 vUv0;",
+        "",
+        "void main(void)",
+        "{",
+        graphicsDevice.webgl2 ? "    vUv0 = vec2((gl_VertexID << 1) & 2, gl_VertexID & 2);" : "   vUv0 = (aPosition.xy + 1.0) * 0.5;",
+        graphicsDevice.webgl2 ? "    gl_Position = vec4(vUv0 * 2.0f - 1.0f, 0.0f, 1.0f);" : "    gl_Position = vec4(aPosition, 0.0, 1.0);",
+        "}"
+    ].join("\n");
+
```

Comment:
I feel like this would be simpler as two different shaders?

Reply:
Ah, oh my. Thanks for catching this, I think this shader won't work.

Reply:
Or just use webgl1 version for both platforms? Not sure we win much with custom webgl2 shader

Reply:
Yeah I have no idea why the fancy bit ops on webgl2. The headers are still different though between 1 and 2.

Reply:
This was a part of another piece of work I did and accidentally carried over in this PR.